travisBuildScript.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+set -e
+
+mkdir $BUILD_DIR
+cd $BUILD_DIR
+
+CMAKE_FLAGS="-DCMAKE_CXX_COMPILER=$COMPILER"
```

Comment:
@dsodintsov I've faced some problems with the clang build on the Travis. After some googling, I end up with the solution needing to pass a flag to the compiler. I've figured it's not a CMake problem, but more of a correct clang setup. So currently I'm just setting it in the .travis.yml.